static/app/views/issueList/customViewsHeader.tsx

```diff
@@ -292,34 +293,48 @@ function CustomViewsIssueListHeaderTabsContent({
   }, [navigate, organization.slug, query, sort, viewId, tabListState]);
 
   // Update local tabs when new views are received from mutation request
-  useEffect(() => {
-    const currentViewId = viewId;
+  useEffectAfterFirstRender(() => {
```

Comment:
Changed this to a `useEffectAfterFirstRender` because this useEffect should only be triggered _after_ the user has made an action that alters the tab state, which isn't possible before the component renders.